recipes-extended/wxwidgets/wxwidgets_3.0.3.1.bb

```diff
@@ -4,45 +4,37 @@ HOMEPAGE = "http://www.wxwidgets.org"
 LICENSE = "WXwindows"
 LIC_FILES_CHKSUM = "file://docs/licence.txt;md5=18346072db6eb834b6edbd2cdc4f109b"
 
-DEPENDS = "webkit-gtk gstreamer gtk+ jpeg tiff libpng zlib expat libxinerama libglu"
+DEPENDS = "webkitgtk gstreamer gtk+ jpeg tiff libpng zlib expat libxinerama libglu"
 
-SRC_URI = "${SOURCEFORGE_MIRROR}/wxwindows/wxWidgets-${PV}.tar.bz2"
-SRC_URI[md5sum] = "e98c5f92805493f150656403ffef3bb0"
-SRC_URI[sha256sum] = "b74ba96ca537cc5d049d21ec9ab5eb2670406a4aa9f1ea4845ea84a9955a6e02"
+SRC_URI = "https://github.com/wxWidgets/wxWidgets/releases/download/v${PV}/wxWidgets-${PV}.tar.bz2"
+SRC_URI[md5sum] = "694ed5f5ea1597e06a9adc6f347d8929"
+SRC_URI[sha256sum] = "3164ad6bc5f61c48d2185b39065ddbe44283eb834a5f62beb13f1d0923e366e4"
 
 S = "${WORKDIR}/wxWidgets-${PV}"
 
-inherit autotools-brokensep pkgconfig binconfig
-
-EXTRA_AUTORECONF = " -I ${S}/build/aclocal"
-EXTRA_OECONF = "  --with-opengl \
-                 --without-sdl \
-                 --enable-mediactrl=yes \
-                 --enable-webviewwebkit=yes \
-                 --disable-gpe \
-                 --disable-visibility \
-                 --disable-rpath \
-               "
+#inherit autotools pkgconfig binconfig
```

Comment:
the commented-out lines can just be dropped; it looks like some left-overs from your development and testing.